app-modules/prospect/tests/Prospect/EditProspectTest.php

```diff
@@ -220,26 +218,3 @@
         ->full_name->toBe($student->full_name)
         ->email->toBe($student->email);
 });
-
-test('disassociate student from prospect', function () {
```

Comment:
@ankit-canyon why was this test removed? I think it is still needed and possible to do right?

Reply:
Hi @Orrison 

As EditProspect page will be forbidden in case of prospect is converted to student. Hence I removed the test case and added below test case:-

`edit page is forbidden when prospect is converted to student`

Thank you.

Reply:
@ankit-canyon Ah! I see the test is still on the View page testing. Got it. Thank you!